Dockerfile

```diff
@@ -167,7 +167,8 @@ RUN wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb \
   && echo "$ELIXIR_CHECKSUM  Precompiled.zip" | sha512sum -c - \
   && unzip -d /usr/local/elixir -x Precompiled.zip \
   && rm -f Precompiled.zip erlang-solutions_1.0_all.deb \
-  && mix local.hex --force
+  && mix local.hex --force \
+  && mix archive.install hex nerves_bootstrap --force
```

Comment:
Since we're already doing this in the build script, I think we can get rid of it here right?

Reply:
if this build script is used during regular builds as well (and not only test ones) I think it is safe to remove it 👍

Reply:
Yep, it is 👍 https://github.com/dependabot/dependabot-core/blob/6a682e7381893edc7df03961662db5494cd0dc2c/Dockerfile#L202

Reply:
Updated.